pkg/services/sqlstore/migrations/ualert/tables.go

```diff
@@ -36,6 +36,14 @@ func AddTablesMigrations(mg *migrator.Migrator) {
 
 	AddAlertmanagerConfigHistoryMigrations(mg)
 	ExtractAlertmanagerConfigurationHistoryMigration(mg)
+
+	mg.AddMigration("add applied_at column to alert_configuration_history", migrator.NewAddColumnMigration(migrator.Table{Name: "alert_configuration_history"}, &migrator.Column{
+		Name: "applied_at", Type: migrator.DB_Int, Nullable: true,
+	}))
+
+	mg.AddMigration("add applied_at column to alert_configuration", migrator.NewAddColumnMigration(migrator.Table{Name: "alert_configuration"}, &migrator.Column{
+		Name: "applied_at", Type: migrator.DB_Int, Nullable: true,
+	}))
```

Comment:
Do we need this column in both tables? We only ever read it from the history one.